packages/presenters/GroupPresenter.js

```diff
@@ -51,13 +51,15 @@ function shouldWelcomeResolver (group, currentUser) {
 
 function contextWidgetsResolver (group, currentUser) {
   if (isContextGroupSlug(group.slug)) {
-    return getStaticMenuWidgets({
-      isPublicContext: group.slug === PUBLIC_CONTEXT_SLUG,
-      isMyContext: group.slug === MY_CONTEXT_SLUG,
-      profileUrl: profileUrlResolver(currentUser)
-    })
+    return [
```

Comment:
I don't get what is going on here?

Is this just something to handle a contextGroup having its contextWidgets resolved several times?

Reply:
Oh, this is to have a home widget?

Reply:
Yes, I think there are comments about this in the code. For mobile there is an added static homeWidget for the static my and public groups.